praas-app-ionic/src/components/Conduit/options.ts

```diff
@@ -1,5 +1,5 @@
 export const conduitTypes = [
   { value: 'Google Sheets', label: 'Google Sheets' },
```

Comment:
As I said in the above comments, as these are constant, possibly the FE value mapping could be also a part of this

Reply:
The backend is validating the `suriType` which should match with the `value` fields here. 
Ideally, value should be `kebab_case` or `camelCase` values which would make it more meaningful and cleaner.

Mapping them here would probably add more code which actually is not necessary, also needing to remap the `suriType` field back to what API expects on form submit.
In future steps, the prefix list and regex should be coming from a backend api to make it more robust to changes.